### ISSUE TITLE
Users cannot view their own assets

### ISSUE THREAD

**Issue by dangngoctrung1806** [grokability/snipe-it#11902]
### Is your feature request related to a problem? Please describe.

Users cannot view their own assets. Admin checkout asset to user, but user login into no show their own assets.

### Describe the solution you'd like

User can view their own assets

**Comment by snipe** [comment]
I don't think this is a bug - or at least it's not one I can reproduce. Do you have full multiple company support enabled? If so, can you confirm that the user and the assets assigned to them belong to the same company?

**Comment by snipe** [comment]
Hi there - We haven't heard back in a bit, so I'm going to close this ticket for now, but will re-open it if you're still having issues.

![hackerman-hacking](https://user-images.githubusercontent.com/197404/31936003-27a7f6d8-b865-11e7-8f5e-eb03062925f4.gif)
